guides/v2.2/config-guide/redis/redis-pg-cache.md

```diff
@@ -81,75 +49,39 @@ where
-<tr>
-<td>page-cache-redis-db</td>
-<td>database</td>
-<td>
-<p>Required if you use Redis for both the default and full page cache. You must specify the database number of one of the caches; the other cache uses <code>0</code> by default.</p>
-<p><strong>Important</strong>: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2. </p>
-</td>
-<td>0</td>
-</tr>
-<tr>
-<td>page-cache-redis-password</td>
-<td>password</td>
-<p>Configuring a <strong>Redis password</strong> enables one of its built-in security features: the auth command, which requires clients to authenticate to access the database. The <strong>password</strong> is configured directly in Redis's configuration file, <code>/etc/redis/redis.conf</code>, which you should still have open from the previous step.</p>
-<td>0</td>
-</tr>
-<tr>
-<td>page-cache-redis-compress-data</td>
-<td>compress_data</td>
-<td>Required only for the full page cache. Set to <code>1</code> to compress the full page cache. Redis chooses a compression algorithm in the following order, based on availability: <a href="https://github.com/google/snappy">snappy</a>, <a href="https://github.com/Cyan4973/lz4">l4z</a>, or <a href="http://oldhome.schmorp.de/marc/liblzf.html">lzf</a>. If none of them available, Redis uses gzip.</td>
-<td>0</td>
-</tr>
-</tbody>
-</table>
-
-## Example command
-
-The following example enables Redis page caching, sets the host to `redis.example.com` and assigns the database number to 1. All other parameters are set to the default value.
-
-`bin/magento setup:config:set --page-cache=redis --page-cache-redis-server=redis.example.com --page-cache-redis-db=1`
+|Command line parameter|Parameter|Meaning|Default value|
+|--- |--- |--- |--- |
+|cache-backend-redis-server|server|Fully qualified hostname, IP address, or an absolute path to a UNIX socket. The default value of 127.0.0.1 indicates Redis is installed on the Magento server.|127.0.0.1|
+|cache-backend-redis-port|port|Redis server listen port|6379|
+|cache-backend-redis-db|database|Required if you use Redis for both the default and full page cache. You must specify the database number of one of the caches; the other cache uses 0 by default.
+Important: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
+|cache-backend-redis-password|password|Configuring a Redis password enables one of its built-in security features: the auth command, which requires clients to authenticate to access the database. The password is configured directly in Redis's configuration file, /etc/redis/redis.conf, which you should still have open from the previous step.||
```

Comment:
It's usually a good idea to add some inline styling at the end of each table to auto size the columns. For example:

`|cache-backend-redis-password|password|Configuring a Redis password enables one of its built-in security features: the auth command, which requires clients to authenticate to access the database. The password is configured directly in Redis's configuration file, /etc/redis/redis.conf, which you should still have open from the previous step.||{:style="table-layout:auto;"}`

See the Style Guide for more info: https://wiki.corp.magento.com/display/MDOC/Style+guide+-+formatting#Styleguide-formatting-Tables

Reply:
"is configured" might read better "Configure the password using the Redis `/etc/redis/redis.conf` configuration file..." to avoid Redis being possessive and the sentence being passive.